### ISSUE TITLE
[IntelliJ][ReportedByUser] It occurrs  an error when remotely debug with HDI4

### ISSUE THREAD

**Issue by v-xiaocw** [Microsoft/azure-tools-for-java#2866]
IntelliJ build version: 2018.3.1 IC-183.4588.27
OS: Windows 10
JDK: JetBrains s.r.o 1.8.0_152-release
Plugin version: 3.19.0-2018.3
Repro Steps:
1.Create a new HDInsight project
2.Create a Run config file  
select spark23-hdi4-yan-test and default storage account
3.Set a breakpoint
4.Click the debug button
Result:
```
Additional Info: None
Parent component: 
Job run failed with unexpected RuntimeException: ReferenceError: "resourceRequestsTableData" is not defined. (https://spark23-hdi4-yan-test.azurehdinsight.net/yarnui/static/jquery/jquery-3.3.1.min.js#2)

======= EXCEPTION START ========
Exception class=[net.sourceforge.htmlunit.corejs.javascript.JavaScriptException]
com.gargoylesoftware.htmlunit.ScriptException: ReferenceError: "resourceRequestsTableData" is not defined. (https://spark23-hdi4-yan-test.azurehdinsight.net/yarnui/static/jquery/jquery-3.3.1.min.js#2)
	at com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine$HtmlUnitContextAction.run(JavaScriptEngine.java:898)
	at net.sourceforge.htmlunit.corejs.javascript.Context.call(Context.java:637)
	at net.sourceforge.htmlunit.corejs.javascript.ContextFactory.call(ContextFactory.java:518)
	at com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine.callFunction(JavaScriptEngine.java:827)
	at com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine.callFunction(JavaScriptEngine.java:799)
	at com.gargoylesoftware.htmlunit.html.HtmlPage.executeJavaScriptFunctionIfPossible(HtmlPage.java:2470)
	at com.gargoylesoftware.htmlunit.javascript.background.JavaScriptFunctionJob.runJavaScript(JavaScriptFunctionJob.java:52)
	at com.gargoylesoftware.htmlunit.javascript.background.JavaScriptExecutionJob.run(JavaScriptExecutionJob.java:102)
	at com.gargoylesoftware.htmlunit.javascript.background.JavaScriptJobManagerImpl.runSingleJob(JavaScriptJobManagerImpl.java:424)
	at com.gargoylesoftware.htmlunit.javascript.background.DefaultJavaScriptExecutor.run(DefaultJavaScriptExecutor.java:155)
	at java.lang.Thread.run(Thread.java:745)
Caused by: net.sourceforge.htmlunit.corejs.javascript.JavaScriptException: ReferenceError: "resourceRequestsTableData" is not defined. (https://spark23-hdi4-yan-test.azurehdinsight.net/yarnui/static/jquery/jquery-3.3.1.min.js#2)
	at net.sourceforge.htmlunit.corejs.javascript.Interpreter.interpretLoop(Interpreter.java:1009)
	at net.sourceforge.htmlunit.corejs.javascript.Interpreter.interpret(Interpreter.java:800)
	at net.sourceforge.htmlunit.corejs.javascript.InterpretedFunction.call(InterpretedFunction.java:105)
	at net.sourceforge.htmlunit.corejs.javascript.ContextFactory.doTopCall(ContextFactory.java:416)
	at com.gargoylesoftware.htmlunit.javascript.HtmlUnitContextFactory.doTopCall(HtmlUnitContextFactory.java:292)
	at net.sourceforge.htmlunit.corejs.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3264)
	at com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine$4.doRun(JavaScriptEngine.java:820)
	at com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine$HtmlUnitContextAction.run(JavaScriptEngine.java:883)
	... 10 more
JavaScriptException value = ReferenceError: "resourceRequestsTableData" is not defined.
== CALLING JAVASCRIPT ==
  function () {
      throw t;
  }
======= EXCEPTION END ========
```

**Comment by wezhang** [comment]
@t-rufang , could you help have a look?

**Comment by t-rufang** [comment]
Looking.

**Comment by wezhang** [comment]
The root cause is our browser client `htmlunit` only support jquery 1.x version, the cluster has upgraded the Yarn to 3.1 with jquery 3.3.1.

**Comment by t-rufang** [comment]
Fixed in #2952

**Comment by v-xiaocw** [comment]
Fixed in build 1067
